### PR TITLE
[develop] Updating ConfigWorkflow.rst file to reflect changes to Config defaults

### LINKED ISSUE #1156: [develop]: Updated ConfigWorkflow.rst to reflect changes to config_defaults.yaml (PI14)
Periodically, documentation needs to be updated to reflect the most recent changes to the repository. Generally, updates include:

Edits to ConfigWorkflow.rst to align it with config_defaults.yaml

Solution
Update documentation to reflect any recent changes.

### PR DESCRIPTION
**Pull request by jdkublnick**
DESCRIPTION OF CHANGES:
Updated ConfigWorkflow.rst to reflect recent changes to config_defaults.yaml in order to keep documentation up to date.

Type of change
[X ] This change requires a documentation update

TESTS CONDUCTED:
None required for documentation. The docs build successfully on my fork and can be viewed at: https://ufs-srweather-app-joshua-kublnick.readthedocs.io/en/text-us-404/

DEPENDENCIES:
N/A

DOCUMENTATION:
All documentation

ISSUE:
Issue https://github.com/ufs-community/ufs-srweather-app/issues/1132

CHECKLIST
[X ] My code follows the style guidelines in the Contributor's Guide
[X ] I have performed a self-review of my own code using the Code Reviewer's Guide
I have commented my code, particularly in hard-to-understand areas
[X ] My changes need updates to the documentation. I have made corresponding changes to the documentation
My changes do not require updates to the documentation (explain).
[X ] My changes generate no new warnings
New and existing tests pass with my changes
[ X] Any dependent changes have been merged and published
LABELS (optional):
A Code Manager needs to add the following labels to this PR:

[X ] documentation

### CODE REVIEW

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst [hunk 1]
```diff
@@ -14,7 +14,7 @@ The following is a list of the parameters in the ``config_defaults.yaml`` file.
 
 .. _user:
 
-USER Configuration Parameters
+USER-related Configuration Parameters
 =================================
```
Comment (MichaelLueken) [inline doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst]: The Doc Tests GitHub Action is failing due to the title underline being too short.  Please increase the size of the underline as appropriate.

**Review by MichaelLueken — CHANGES_REQUESTED** [review]
@jdkublnick -

Thanks for adding these changes into the `ConfigWorkflow.rst` file!  The `Doc Tests` GitHub Action test is failing due to the title underline being too short on line 18.  Please increase the size of the underline as appropriate.

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst (hunk 1 above)
Comment (jdkublnick) [reply doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst]: I fixed this, thank you for pointing that out

**Review by MichaelLueken — APPROVED** [review]
@jdkublnick -

Thanks for making the modification to allow the `Doc Tests` to successfully pass!

Approving now.

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst [hunk 2]
```diff
@@ -14,8 +14,8 @@ The following is a list of the parameters in the ``config_defaults.yaml`` file.
 
 .. _user:
 
-USER Configuration Parameters
-=================================
+USER-related Configuration Parameters
```
Comment (gspetro-NOAA) [inline doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst]:
```suggestion
USER-Related Configuration Parameters
```

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst [hunk 3]
```diff
@@ -360,6 +360,9 @@ Set File Name Parameters
 ``AQM_RC_TMPL_FN``: (Default: "aqm.rc")
    Template file name of resource file for NOAA Air Quality Model (AQM). 
 
+``FIRE_NML_FN_``: (Default: "namelist.fire")
```
Comment (gspetro-NOAA) [inline doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst]:
```suggestion
``FIRE_NML_FN``: (Default: "namelist.fire")
```

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst [hunk 4]
```diff
@@ -390,6 +393,8 @@ Set File Path Parameters
 ``AQM_RC_TMPL_FP``: (Default: ``'{{ [user.PARMdir, AQM_RC_TMPL_FN]|path_join }}'``) 
    Path to the ``AQM_RC_TMPL_FN`` file. 
 
+``FIRE_NML_BASE_FP``: (Default: ``'{{ [user.PARMdir, FIRE_NML_FN]|path_join }}'``)
+   Path to the ``FIRE_NML_BASE_FP`` file.
```
Comment (gspetro-NOAA) [inline doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst:397]:
```suggestion
   Path to the ``FIRE_NML_BASE`` file.
```

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst [hunk 5]
```diff
@@ -411,6 +416,9 @@ This section contains files and paths to files that are staged in the experiment
 ``FV3_NML_STOCH_FP``: (Default: ``'{{ [EXPTDIR, [FV3_NML_FN, "_stoch"]|join ]|path_join }}'``)
    Path to a namelist file that includes stochastic physics namelist parameters. 
 
+``FIRE_NML_FP``: (Default: ``'{{ [EXPTDIR, FIRE_NML_FN]|path_join }}'``)
+   Path to the ``FIRE_NML_FP`` in the experiment directory.
```
Comment (gspetro-NOAA) [inline doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst]:
```suggestion
   Path to the ``FIRE_NML_FN`` file in the experiment directory.
```

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst (hunk 4 above)
Comment (gspetro-NOAA) [inline doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst]:
```suggestion
   Path to the ``FIRE_NML_FN`` file in the ``parm`` directory.
```

**Comment by gspetro-NOAA** [comment]
In the RST, `FIRE_IGNITION_START_TIME` is listed twice. I think the developer made a copy-paste error, and `FIRE_IGNITION_START_TIME (Default: 7000)` should be changed to `FIRE_IGNITION_END_TIME (Default: 7000)`. Other than that and the few changes above, looks good!

File: doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst [hunk 6]
```diff
@@ -2143,7 +2151,7 @@ Non-default parameters for the Community Fire Behavior Model (CFBM) in SRW are s
 .. note::
    If ``FIRE_NUM_IGNITIONS > 1``, the following variables should be lists with one entry for each ignition
 
-``FIRE_IGNITION_ROS`` (Default: 0.0)
+``FIRE_IGNITION_ROS`` (Default: 0.5)
```
Comment (gspetro-NOAA) [inline doc/UsersGuide/CustomizingTheWorkflow/ConfigWorkflow.rst]:
```suggestion
``FIRE_IGNITION_ROS`` (Default: 0.05)
```

**Review by gspetro-NOAA — APPROVED** [review]
@jdkublnick has made all of my requested changes.

**Pull request merged**